spiceaidocs/docs/components/data-connectors/mssql.md

```diff
@@ -16,21 +25,51 @@ datasets:
 
 ## Configuration
 
+### `from`
+
+The `from` field takes the form `mssql:path.to.my_dataset` where `path.to.my_dataset` is the fully-qualified table name in the SQL server.
```

Comment:
Let's use the vocabulary that MSSQL calls their tables, which is `database.schema.table`

Reply:
Fixed, thank you!